### PR TITLE
Add `clean` subcommand to ansible-bender

### LINKED ISSUE #50: DB cleanup
Since I'm still adding more and more stuff to the DB, there should be a sensible way to clean it up: e.g., remove entries from DB for images which are no longer present or something like that.

### PR DESCRIPTION
**Pull request by kmehant**
Fixes #50

### CODE REVIEW

**Comment by kmehant** [comment]
@TomasTomecek Shall we even add a new command `rm` to remove specific entries by build id.

**Comment by kmehant** [comment]
> Nicely done!
> 
> I'm a little concerned about performance here since `podman inspect` takes some time if you do it for every image it may take some time.
> 
> How about we added a new command to clean the database?
> 
> ```
> $ ansible-bender clean
> Cleaning images from database which are no longer present on the disk...
> ...
> Done!
> ```

@TomasTomecek that would be a awesome fix in terms of performance, would make those changes

**Comment by TomasTomecek** [comment]
awesome! also please rebase your PR against master, since I finally added CI to bender and it's passing for master now

**Comment by kmehant** [comment]
@TomasTomecek I have added `clean` subcommand, perhaps shall we add another subcommand `rm` to remove specific build entries from ab database along with their container images. Please let me know if it is good to add `rm`

**Comment by TomasTomecek** [comment]
I wouldn't worry about `rm` command, that seems too specific - clean is just enough to be usable by anyone. I'd say that an average user would be confused of such rm command.

File: ansible_bender/cli.py [hunk 1]
```diff
@@ -347,6 +355,20 @@ def _push(self):
...
+        for b in builds:
+            try:
+                run_cmd(["podman", "inspect", b.target_image],
+                        return_output=False, log_output=False)
+            except subprocess.CalledProcessError:
+                self.app.remove_build(b.build_id)
+                print(f"Build entry with ID {b.build_id} has been removed from DB as it no longer has it's corresponding image")
+                continue
+            finally:
+                print("Done!")
```
Comment (TomasTomecek) [inline ansible_bender/cli.py]: This will print "Done" for every build in the DB - let's do "Done" after the for-loop.

**Comment by kmehant** [comment]
I will push changes in moment to fix some testing errors just got it through CI pipeline.

**Comment by kmehant** [comment]
@TomasTomecek I went through the packit logs and found that there was an assertion error because of this
`
assert "Build entry with ID" in out
`
so finally fixed it. I believe it is ready to merge now 😄

**Comment by TomasTomecek** [comment]
Awesome!

**Pull request merged**